tests/behavior/transforms/remap.toml

```diff
@@ -1365,3 +1365,24 @@
       "namespace.equals" = "zork"
       "host.equals" = "ook"
       "type.equals" = "counter"
+
+[transforms.remap_function_encode_json]
+  inputs = []
+  type = "remap"
+  source = """
+    .a = encode_json(.a)
```

Comment:
I think this could be more useful if you do:

`.a = encode_json(parse_json(.a))`

Then you are actually encoding some more complex data than Strings.